SPO/README.md

```diff
@@ -55,6 +55,13 @@ Please find the instructions [here](https://docs.microsoft.com/en-us/azure/azure
     ```
 
     b. Drop a `<script>` tag on your extension head.
+
+    > ***Note:*** You may refer to the main README page [here](https://github.com/microsoft/ApplicationInsights-JS#snippet-setup-ignore-if-using-npm-setup) for the most updated snippet version. The snippet below is slightly different:
+    > 1. Add an extra `\` for all `\*` cases to excape the second `\`.
+    > 2. Use `"https://az416426.vo.msecnd.net/scripts/b/ai.2.gbl.min.js"` as the SDK URL source.
+    > You can also refer to [Snippet configuration options](https://github.com/Microsoft/ApplicationInsights-JS#snippet-configuration-options) page to add optional configurations for the snippet below.
+    
```

Comment:
hi @MSNev I updated to add above Note, does it clear enough to cover your suggestions? thanks~